README.md

```diff
@@ -183,6 +183,18 @@ Thanks to everyone who helped make `Carch` better. Your contributions are greatl
    </a>
 
 </div>
+<br>
+<div align="center">
+ <a href="">
+<img src="https://github.com/harilvfs/assets/blob/main/carch/arrow.png" width="30"/>
+</a>
+</div>
+
+<div align="center">
```

Comment:
You don’t need to create a separate div for the back-to-top (BTT) PNG. You can place it within the upper div by using a <br> tag below the arrow PNG img src.